converter/shieldhit/parser.py

```diff
@@ -270,25 +279,28 @@ def _parse_world_zone(self, json: dict) -> None:
                     # world zone defined by bounding figure and contained zones
                     figures_operators=[operation],
                     # the material of the world zone is usually defined as vacuum
-                    material=self._get_material_id(world_zone["materialUuid"])))
+                    material=material))
 
-        # Add the figure that will serve as a black hole wrapper around the world zone
-        black_hole_figure = solid_figures.parse_figure(world_zone)
-        # Make the figure slightly bigger. It will form the black hole wrapper around the simulation.
-        black_hole_figure.expand(1.)
+        if material != DefaultMaterial.BLACK_HOLE.value:
```

Comment:
Add some comment explaining why black hole is added automatically only if `material` is different from black hole